docs/source/examples/simple-landscape-qiskit.myst

```diff
@@ -0,0 +1,250 @@
+
+from mitiq.zne import mitigate_executor
+from mitiq.zne.inference import RichardsonFactory
+```
+
+## Defining the ideal variational circuit in Qiskit 
+
++++
+
+We define a function which returns a simple two-qubit variational circuit depending on a single parameter  $\gamma$ ("gamma").
+
+```{code-cell} ipython3
+def variational_circuit(gamma: float) -> QuantumCircuit:
+    """Returns a two-qubit circuit for a given variational parameter.
+
+    Args:
+        gamma: The variational parameter.
+
+    Returns:
+        The two-qubit circuit with a fixed gamma.
+    """
+    circuit = QuantumCircuit(2)
+    circuit.rx(gamma, 0)
+    circuit.cnot(0, 1)
+    circuit.rx(gamma, 1)
+    circuit.cnot(0, 1)
+    circuit.rx(gamma, 0)
+    
+    return circuit
+```
+
+We can visualize the circuit for a particular $\gamma$ as follows.
+
+```{code-cell} ipython3
+circuit = variational_circuit(gamma=np.pi)
+circuit.draw()
+```
+
+## Defining the executor functions with and without noise
+To use error mitigation methods in Mitiq, we define an executor function which computes the expectation value of a simple Hamiltonian $H=Z \otimes Z$, i.e., Pauli-$Z$ on each qubit. To compare to the noiseless result, we define both a noiseless and a noisy executor below.
+
+```{code-cell} ipython3
+# observable to measure
+z = np.diag([1, -1])
+hamiltonian = np.kron(z, z)
+
+def noiseless_executor(circuit: QuantumCircuit) -> float:
+    """Simulates the execution of a circuit without noise.
+
+    Args:
+        circuit: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ observable.
+    """
+    # avoid mutating the input circuit
+    circ = circuit.copy()
+    circ.save_density_matrix()
+
+    # execute experiment without noise
+    job = qiskit.execute(
+        experiments=circ,
+        backend=qiskit.Aer.get_backend("aer_simulator_density_matrix"),
+        noise_model=None,
+        # we want all gates to be actually applied,
+        # so we skip any circuit optimization 
+        optimization_level=0,
+        shots=1,
+    )
+    rho = job.result().data()["density_matrix"]
+
+    expectation = np.real(np.trace(rho @ hamiltonian))
+    return expectation 
+    
+
+
+# strength of noise channel
+noise_level = 0.04
+
+def executor_with_noise(circuit: QuantumCircuit) -> float:
+    """Simulates the execution of a circuit with depolarizing noise.
+
+    Args:
+        circuit: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ hamiltonian.
+    """
+    # avoid mutating the input circuit
+    circ = circuit.copy()
+    circ.save_density_matrix()
+    
+    # Initialize qiskit noise model. In this case a
+    # depolarizing noise model with the same noise strength for each
+    # gate of the standard IBM basis
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(
+        depolarizing_error(noise_level, 1), ["u1", "u2", "u3"]
+    )
```

Comment:
I think this instruction has no effect since `"u1", "u2", "u3"` gates are not in the circuit.
Possible alternative solutions (I haven't a strong preference) are:

1. Add noise to rx gates.
```
   noise_model.add_all_qubit_quantum_error(
        depolarizing_error(noise_level, 1), ["rx"]
    )
```
2. Add noise to rx gates but smaller than noise of CNOT gates.
```
   noise_model.add_all_qubit_quantum_error(
        # We assume rx noise is 10 times weaker than CNOT noise. 
        depolarizing_error(0.1 * noise_level, 1), ["rx"]
    )
```
3. Only add noise to CNOT gates. Maybe mention it in the docstring.

Reply:
Oh yeah that makes more sense given the circuit. I was thinking noise on `"u1", "u2", "u3"' would result in the same noisy energy landscape because `qiskit.execute()` transpiles the input circuit to the noise model gate set. But that's confusing/opaque.

I changed it to solution 1 (same noise level on rx and cnot) since I think this is closest to the Cirq example.

Reply:
>  `qiskit.execute()` transpiles the input circuit to the noise model gate set.
> 
Ah, interesting, I didn't know that. 

Anyway, the new version is more explicit and clear.  Thanks!